src/fixtures/assets.js

```diff
@@ -54,6 +54,23 @@ export const initialAssets = [
     isDefault: true,
     balance: 0,
   },
+  {
+    address: '0x9471ea41772925e3ff477772681740c9f520d3af',
```

Comment:
What would this address be? :thinking:

Reply:
this prop doesn't get used. If used in tests it helps as mock data